src/dom/progress-watcher.ts

```diff
@@ -0,0 +1,165 @@
+
+import { is } from '../is/is';
+import { mathf } from '../mathf/mathf';
+
+export interface ProgressWatcherItem {
+    range: number | Array<number>;
```

Comment:
I think Google's TS style guide suggests using `number[]` for array types.

Reply:
Done